k8s/app-python/templates/statefulset.yaml

```diff
@@ -29,9 +29,18 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       volumes:
+        - name: app-python-init
+          emptyDir: {}
         - name: {{ .Values.volumes.config.name }}
           configMap:
             name: {{ .Values.volumes.config.name }}
+      initContainers:
+        - name: app-python-init
+          image: python:3.10
```

Comment:
not sure why you really need peethon here since you're using just `curl` which should be available in busybox or whatever

if it works :trollface: don't touch it ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)